ui/component/fileRender/view.jsx

```diff
@@ -113,122 +109,58 @@ class FileRender extends React.PureComponent<Props, State> {
   }
 
   renderViewer() {
-    const { mediaType, currentTheme, claim, contentType, downloadPath, fileName, streamingUrl, uri } = this.props;
-    const fileType = fileName && path.extname(fileName).substring(1);
-
-    // Ideally the lbrytv api server would just replace the streaming_url returned by the sdk so we don't need this check
-    // https://github.com/lbryio/lbrytv/issues/51
-    const source = IS_WEB ? generateStreamUrl(claim.name, claim.claim_id) : streamingUrl;
-
-    // Human-readable files (scripts and plain-text files)
-    const readableFiles = ['text', 'document', 'script'];
-
-    // Supported mediaTypes
-    const mediaTypes = {
```

Comment:
media and content types should not need to be messed with for the most part

just use renderMode